android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -66,6 +66,33 @@ public void run() {
         });
     }
 
+    /*
+     * Returns all ccontacts matching name
+     */
+    @ReactMethod
+    public void getContactsMatchingString(final String searchString, final Callback callback) {
+        getAllContactsMatchingString(searchString, callback);
```

Comment:
You know, I weighed the same thing and opted to keep it like `getAllContacts` and `getAllContactsWithoutPhotos` in case we needed a similar API for the filtered contact. Happy to remove if you disagree.